### ISSUE TITLE
Create new Optimism subgraph post-regenesis

### ISSUE THREAD

**Issue by willhennessy** [Uniswap/v3-info#178]
The optimism info site and Add Liquidity in App are both dead since the Optimism Regenesis. Let's create a new subgraph to get this working again

**Comment by willhennessy** [comment]
* big scope 
* some ideas to get the current state, but hard is getting historical state
* next step: design a solution with the Graph team

**Comment by willhennessy** [comment]
note: will try a subgraph deploy today
if that doesn't work, let's at last resume showing current data from post-regenesis and tackle historical data separately

**Comment by DominicRomanowski** [comment]
Hey 👋 has a subgraph been deployed with data from post-regenesis?

**Comment by willhennessy** [comment]
Update from Ian:  in progress. Currently his top task. Lyra was unblocked, but we still need to fix this.

Optimistically we can get something working this week with ~2 days of work.

**Comment by K-Ho** [comment]
Let me know if there's anything I can do to support here! <3

**Comment by nejcb1** [comment]
same here - can we help?

**Comment by ianlapham** [comment]
@K-Ho @nejcb1 Hey guys here some info on the current state of optimism subgraph data 

Was able to succesfuly get post-regenesis data but manually adding pre-regenesis pools. 
The subgraph endpoint for that subgraph is [here](https://thegraph.com/hosted-service/subgraph/ianlapham/optimism-post-regenesis)

The code for that subgraph is [here](https://github.com/Uniswap/v3-subgraph/tree/ian/optimism-fix)

The main issue is that the graph doesnt index optimism blocks for that event anymore - I know that the graph has a snapshot of the data. So i think the solution for getting pre-regenesis data is to manually set properties on the entities based on that snapshot

**Comment by willhennessy** [comment]
thanks ian! I'll mark this issue as complete and open a new issue to add pre-regenesis data in the future: https://github.com/Uniswap/v3-info/issues/197

**Comment by Neokun** [comment]
Hello @ianlapham , can I ask for the snapshot on pre-regenesis?
Currently, the optimism-post-regenesis subgraph is having wrong/missing tick data of many pools (for example the pool `0x03af20bdaaffb4cc0a521796a223f7d85e2aac31`)
And I need the full tick list data of the pool. Or is there any fix to get all tick data of the pool?
